scripts/get-akse.sh

```diff
@@ -79,7 +79,7 @@ checkDesiredVersion() {
   elif type "wget" > /dev/null; then
     TAG=$(wget -q -O - $release_url | awk '/\/tag\//' | grep -v no-underline | grep "<a href=\"/Azure/aks-engine/releases" | head -n 1 | cut -d '"' -f 2 | awk '{n=split($NF,a,"/");print a[n]}' | awk 'a !~ $0{print}; {a=$0}')
   fi
-  if [ "x$TAG" == "x" ]; then
```

Comment:
sneaking this one in? :)

Reply:
This release of docker-go-dev also bumped `shellcheck` and it had new two new complaints which I fixed. This one is [SC2268](https://github.com/koalaman/shellcheck/wiki/SC2268), which was a workaround for ancient, non-POSIX-compliant shells that it suggests is just confusing today.

Reply:
Well I lived long enough to see shellcheck take one for the team in the interest of practicality.